README.md

```diff
@@ -23,10 +23,15 @@ branch is our master branch.
 We are using the [Akari Link Shortener](https://waa.ai/) to create a short URL: [https://waa.ai/ucrsv](https://waa.ai/ucrsv). The
 first three characters "ucr" refers to our school's name, and the last 2 characters "sv" are short for "schedule visualizer". We also have [a QR code](http://bradleycai.github.io/ucr-schedule-visualizer/img/waaai-qr.png).
 
+End of life
+-----------
+
+With UCR's recent switch to Banner, we've decided to stop working on UCRSV. We plan to add minimal compatibility for it's schedule, but that'll be the last thing we do for this website. You can read more about it [here](https://bradleycai.github.io/ucr-schedule-visualizer/about.html#end)
```

Comment:
1. Right, thanks
2. I was referring to multiple topics associated to end of life such as the switch to banner, us no longer working on UCRSV, and the addition of Banner compatibility. While "it" doesn't work because it's ambiguous, would "this" work? Something like:
"You can read more about this [here]"
Or I can just refer to the end of life only:
"You can read more about end of life [here]"